stories/core/UserIconGroup.stories.tsx

```diff
@@ -0,0 +1,238 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  getUserColor,
+  Tooltip,
+  UserIcon,
+  UserIconGroup,
+  UserIconGroupProps,
+} from '../../src/core';
+
+export default {
+  component: UserIconGroup,
+  subcomponents: { UserIcon },
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    id: { control: { disable: true } },
+    children: { control: { disable: true } },
+  },
+  title: 'Core/UserIconGroup',
+} as Meta<UserIconGroupProps>;
+
+export const Basic: Story<UserIconGroupProps> = (args) => {
+  const userNames = [
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ace Cash',
+    'Tanner Fraser',
+    'Ashley Miles',
+    'Jean Mullins',
+  ];
+
+  return (
+    <UserIconGroup {...args}>
+      {userNames.map((name) => (
+        <UserIcon
+          size='medium'
+          key={name}
+          abbreviation={name
+            .split(' ')
+            .map((token) => token[0])
+            .join('')}
+          backgroundColor={getUserColor(name)}
+          title={name}
+        />
+      ))}
+    </UserIconGroup>
+  );
+};
+
+Basic.args = {
+  animated: false,
+  iconSize: 'medium',
+};
+
+export const Animated: Story<UserIconGroupProps> = (args) => {
+  const userNames = [
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Jean Mullins',
+    'Ashley Miles',
+  ];
+
+  return (
+    <UserIconGroup animated {...args}>
+      {userNames.map((name) => (
+        <UserIcon
+          size='medium'
+          key={name}
+          abbreviation={name
+            .split(' ')
+            .map((token) => token[0])
+            .join('')}
+          backgroundColor={getUserColor(name)}
+          title={name}
+        />
+      ))}
+    </UserIconGroup>
+  );
+};
+
+Animated.args = {
+  animated: true,
+  iconSize: 'medium',
+};
+
+export const ManyIcons: Story<UserIconGroupProps> = (args) => {
```

Comment:
The original purpose of this story was to show the "9+" variant. Since that has changed to "99+", I think we might need to show that instead.

Reply:
Changed story to show 99+

Reply:
I guess no need for showing two sizes there anymore.

The original intent was that small size would not have the "+", but now they are the same.